### DISCUSSION TITLE
I stopped earning pp again

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22728] [Q&A] [answered]
Since the new update(2023.207.0) I stopped earning pp again even I play new downloaded beatmap, They can't add to my profile again and because of that I stopped going up in Global Ranking. I'm scared becauss my global ranking is severely going down. My pp is stuck at 2,018

My lazer profile: https://lazer.ppy.sh/users/23892930

Here's the log if you ever need it:
[database.log](https://github.com/ppy/osu/files/10819965/database.log)
[network.log](https://github.com/ppy/osu/files/10819966/network.log)
[performance.log](https://github.com/ppy/osu/files/10819967/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10819969/runtime.log)

I hope you fix this ASAP😭

**Comment by Axilotl17** [comment]
your total PP value is decided by your top 100 plays only (but because its weighted realistically you will only see change from top 50), and it looks like youve only set 2 plays in your top 100 since the last update. I think you just haven't set a score worth enough to gain any real PP. 

additionally, PP is still being worked on for lazer, and so you may see many changes to that in the future, so even if this is a real issue, it might get fixed soon!

**Reply by Shirou27** [reply]
What do you mean by weighted? I'm getting confused 

Sorry I can't properly understand you're explanation on how PP works.

**Reply by TamamoNoMae13** [reply]
@Shirou27 https://osu.ppy.sh/wiki/en/Performance_points#weightage-system

**Reply by Shirou27** [reply]
Let's put the weightage thing aside. Other people also experienced this that they stop earning pp after the update.

I saw those concerns in the comment section in this link: https://osu.ppy.sh/home/changelog/lazer/2023.207.0

**Reply by falkon13** [reply]
Yeah I'm getting the same issue as well, last time peppy fixed it with a hotfix, but my pp has frozen across all game modes

**Reply by peppy** [reply]
I've checked and your plays are still getting pp correctly.

**Reply by Shirou27** [reply]
What I mean is the pp from the beatmap I played doesn't added to my profile pp.

Checkout this picture so you can understand what I mean:
![Screenshot_2023-03-03-12-38-27-850-edit_com opera browser](https://user-images.githubusercontent.com/105548541/222633036-63e3a287-0a84-4192-9489-49a469ff6b97.jpg)

**Comment by Shirou27** [comment]
Any update about this?

**Reply by peppy** [reply]
Could you please try submitting another score now and see if it's better?

**Reply by Shirou27** [reply]
I set a score just now but the earned pp still not added to my profile.

**Comment by peppy** [comment, marked as answer]
Should be fixed (via server component restart) and more permanently fixed via https://github.com/ppy/osu-queue-score-statistics/pull/132.

**Reply by Shirou27** [reply]
This means it will be fixed in the next update?

**Reply by peppy** [reply]
Not, it should already be fixed. Please try once more submitting a score that gets you some PP.

**Reply by Shirou27** [reply]
I set a score and I'm earning a pp now, but my pp value decreased but its fine as long as I'm earning pp. 

Thanks for the fix, peppy!❤️

**Comment by rvnexe** [comment]
Me too. I was #‌22 in my country before the updates, and now nothing of my plays are affecting my profile. I've dropped to #‌50
(And dropped from #‌58k to #‌110k in global)
